qa/tasks/cephfs/test_data_scan.py

```diff
@@ -190,6 +200,23 @@ def validate(self):
 
         return self._errors
 
+class BacktracelessFileRemoveLostAndFoundDirectory(Workload):
+    def write(self):
+        self._mount.run_shell(["mkdir", "subdir"])
+        self._mount.write_n_mb("subdir/sixmegs", 6)
+        self._initial_state = self._mount.stat("subdir/sixmegs")
+
+    def flush(self):
+        # Never flush metadata, so backtrace won't be written
+        pass
+
+    def mangle(self):
+        self._filesystem.rados(["-p", self._filesystem.get_metadata_pool_name(), "rm", "4.00000000"])
+        self._filesystem.rados(["-p", self._filesystem.get_metadata_pool_name(), "rmomapkey", "1.00000000", "lost+found_head"])
+
+    def validate(self):
+        # The inode should be linked into lost+found because we had no path for it
```

Comment:
? This test removes lost+found. So this comment doesn't make sense to me.

Reply:
Comment is stale since the most of this test was copied from an existing test. Will remove.